Day3/src/Abtraction.kt

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner
+fun main(args:Array<String>)
+{
+     val sc = Scanner(System.`in`)
+     val book = Books()
+     book.Display()
+
+     val gift:Store =Gifts()
+     println("Please choose your favorite item")
+     println("1.Book 2.Gifts")
+     var choice=sc.nextInt()
+     when(choice)
+     {
+         1 -> book.nameDisplay()
+         2 -> gift.nameDisplay()
+    else ->
+        println("Please choose correct option")
+    }
+}
+abstract class Store
+{
+     var nameStore:String="Awsome"
+    fun Display()
+    {
+        println("Welcome to our Store "+nameStore)
```

Comment:
Maybe you can write this in kotlin way
println("Welcome to our Store $nameStore")